kuflow-temporal-worker/src/main/java/com/kuflow/temporal/worker/jackson/OffsetTimeSerializer.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ * Copyright © 2021-present KuFlow S.L.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.kuflow.temporal.worker.jackson;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import java.io.IOException;
+import java.time.OffsetTime;
+import java.time.format.DateTimeFormatter;
+
+public class OffsetTimeSerializer extends StdSerializer<OffsetTime> {
+
+    private final DateTimeFormatter formatter;
+
+    public OffsetTimeSerializer() {
+        this(DateTimeFormatter.ISO_OFFSET_TIME);
+    }
+
+    public OffsetTimeSerializer(DateTimeFormatter formatter) {
+        super(OffsetTime.class);
+        this.formatter = formatter;
+    }
+
+    @Override
+    public void serialize(OffsetTime value, JsonGenerator gen, SerializerProvider provider) throws IOException {
+        if (value != null) {
```

Comment:
Is this condition possible?

Reply:
I don't know

Reply:
looking into other implementations, ie, EnumSerializer, "value" never can be null.

Reply:
I don't know if it will ever come to NULL, and it saves. I don't know if it has to do with INCLUDE_NON_NULL being enabled or not. But I prefer to leave the IF.

Reply:
The javadoc said that value can't be null, so to avoid confusions,  I prefer to remove the condition:

    /**
     * Method that can be called to ask implementation to serialize
     * values of type this serializer handles.
     *
     * @param value Value to serialize; can <b>not</b> be null.
     * @param gen Generator used to output resulting Json content
     * @param serializers Provider that can be used to get serializers for
     *   serializing Objects value contains, if any.
     */
    public abstract void serialize(T value, JsonGenerator gen, SerializerProvider serializers)
        throws IOException;